src/app/[id]/page.tsx

```diff
@@ -32,10 +32,191 @@ import {
 } from '@/components/ui/sheet';
 import { useBeforeUnload } from '@/utils/hooks/useBeforeUnload';
 import { formSteps } from '@/utils/constants/formSteps';
+import { createAgencyInfoWithServices } from '@/server/actions/Agencies';
+import { AgencyInfoForm } from '@/utils/types';
+import { Service as ServiceModel } from '@/utils/types';
 
 type Inputs = z.infer<typeof FormDataSchema>;
 type Service = z.infer<typeof ServiceSchema>;
 
+const dayMapping: {
```

Comment:
I agree with you that it would probably be helpful to move these conversion functions under `./src/utils/conversions.ts` or something